pyiceberg/io/pyarrow.py

```diff
@@ -387,15 +387,30 @@ def __init__(self, properties: Properties = EMPTY_DICT):
         super().__init__(properties=properties)
 
     @staticmethod
-    def parse_location(location: str) -> Tuple[str, str, str]:
-        """Return the path without the scheme."""
+    def parse_location(location: str, properties: Properties = EMPTY_DICT) -> Tuple[str, str, str]:
+        """Return (scheme, netloc, path) for the given location.
+
+        Uses environment variables DEFAULT_SCHEME and DEFAULT_NETLOC
+        if scheme/netloc are missing.
+        """
         uri = urlparse(location)
-        if not uri.scheme:
-            return "file", uri.netloc, os.path.abspath(location)
-        elif uri.scheme in ("hdfs", "viewfs"):
-            return uri.scheme, uri.netloc, uri.path
+
+        # Load defaults from environment
```

Comment:
nit: these are no longer env vars

Reply:
maybe something like, "If scheme/netloc are missing, fallback to use DEFAULT_SCHEME and DEFAULT_NETLOC properties"